substrate/frame/identity/src/lib.rs

```diff
@@ -922,6 +900,59 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Update the deposits held by `target` for its identity info.
+		///
+		/// Parameters:
+		/// - `target`: The account for which to update deposits.
+		///
+		/// May be called by any signed origin.
+		#[pallet::call_index(16)]
```

Comment:
I mean either one is fine, but this will get in before #1814 
```suggestion
		#[pallet::call_index(15)]
```

Reply:
Done.